CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-# 1.1.0
+# 2.0.0
+
+* Bumped Ion Go to `v1.1.3` allow support for unmarshalling Ion timestamps to Go time objects.
+* Logging interface now provides the LogLevel as a parameter.
+* Expose transaction ID in Transaction interface.
+
+# 1.1.1
```

Comment:
Could we add the date beside the version? (like we do for 1.1.0) 

The released date is: `2021-06-16)`

Reply:
Done.